frontend/src/metabase/query_builder/components/notebook/NotebookCell.jsx

```diff
@@ -51,6 +52,7 @@ NotebookCellItem.defaultProps = {
 
 export const NotebookCellAdd = styled(NotebookCellItem).attrs({
   inactive: ({ initialAddText }) => initialAddText,
+  // eslint-disable-next-line react/display-name
```

Comment:
fyi -- this should be pretty easy to fix by adding a `NotebookCellAdd.displayName = "NotebookCellAdd";` after this value declaration

Reply:
Eslint notifies about the next line with `children: () => ...`. But it is a nice thing to add display names for all components in this file. I've pushed that, thanks